### PR TITLE
Add content hashes to release

### PR DESCRIPTION
**Pull request by dappnodedev**
Release needs to include content hashes, as it is used to enable packages download in case there is no Ethereum node available: https://github.com/dappnode/DNP_CORE/pull/184

With this code, the releases for standard packages will look like:

image

And the releases for multi-variant packages will look like:
image

### CODE REVIEW

**Comment by dappnodedev** [comment]
The best approach would be merging https://github.com/dappnode/DAppNodeSDK/pull/443 and then rebasing before merging this PR

File: src/files/manifest/compactManifestIfCore.ts
Comment (dappnodedev) [inline src/files/manifest/compactManifestIfCore.ts:1]: This file has no changes apart from identation

File: src/providers/github/Github.ts [hunk 1]
```diff
@@ -195,9 +198,9 @@ export class Github {
       body?: string;
       prerelease?: boolean;
     }
-  ): Promise<void> {
+  ): Promise<number> {
```
Comment (dappnodedev) [inline src/providers/github/Github.ts:201]: Only change here: This funcion now returns `releaseID`

File: src/providers/github/Github.ts [hunk 2]
```diff
@@ -214,6 +217,50 @@ export class Github {
...
+            await this.octokit.repos.uploadReleaseAsset({
+              owner: this.owner,
+              repo: this.repo,
+              release_id: releaseId,
+              data: fs.createReadStream(filepath) as any,
+              headers: {
+                "content-type": contentType,
+                "content-length": fs.statSync(filepath).size
+              },
+              name: `${fileNamePrefix || ""}${path.basename(filepath)}`
```
Comment (dappnodedev) [inline src/providers/github/Github.ts:254]: Adds prefix for multivariant packages. For example:

If variant is holesky for Obol package, `content-hash` file name will be `holesky-obol_content-hash`

File: src/tasks/createGithubRelease/subtasks/getCreateReleaseTask.ts [hunk 3]
```diff
@@ -38,72 +35,79 @@ export function getCreateReleaseTask({
...
+
+      task.output = "Uploading assets...";
+      await uploadAssets({ releaseDetailsMap, github, releaseId });
+
     }
   };
 }
 
-async function handleReleaseVariantFiles({
+function prepareGithubReleaseFiles({
```
Comment (dappnodedev) [inline src/tasks/createGithubRelease/subtasks/getCreateReleaseTask.ts:54]: Creates `content-hash` file, compacts maifest and deletes build properties from compose

**Comment by dappnodedev** [comment]
We need to reduce the number of assets uploaded to each release to the minimum before merging this

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**